gallery/src/jsUtilPages/VulnerabilityLevelUtils/VulnerabilityRatingUtilsPage.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxList } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const formatVulnerabilityScoreExampleCode = require('./FormatVulnerabilityScoreExample?raw'),
+    getVulnerabilitySeverityRatingExampleCode = require('./GetVulnerabilitySeverityRatingExample?raw'),
+    vulnerabilityTypesAndRatingsExampleCode = require('./VulnerabilityTypesAndRatingsExample?raw'),
+    vulnerabilitySeverityAndScoreRangeListExampleCode = require('./VulnerabilitySeverityAndScoreRangeListExample?raw');
+
+const VulnerabilityRatingUtilsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        The React Shared Components library includes several simple JavaScript
+        objects and TypeScript types that provide code definitions and mappings to help represent
+        CVSS scores and severity ratings.
+      </NxP>
+      <NxList bulleted>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability (CVSS) Score:</strong>{' '}
+            A number with one decimal point precision between 0.0 and 10.0, inclusive,
+            that describes the severity of a threat. Zero represents no threat at all,
+            and ten represents the most severe of threats.
+          </NxList.Text>
+        </NxList.Item>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability Severity Rating</strong>{' '}
```

Comment:
Would it be worth mentioning that this ranges are defined in CVSS v3 (not defined by Sonatype or one of our products)?
https://nvd.nist.gov/vuln-metrics/cvss

Reply:
nvm, I see `CVSS` is mentioned below